AS_920_923_TTN_VN.yml

```
@@ -0,0 +1,93 @@
+band-id: AS_923
+sub-bands:
+- min-frequency: 920000000
+  max-frequency: 923000000
+duty-cycle: 0.01             # Duty cycle for this sub-band (optional; default: 1)
+max-eirp: 16
```

@viethoa14 I meant, if this applies to the sub-band, it needs to be indented

```suggestion
  duty-cycle: 0.01             # Duty cycle for this sub-band (optional; default: 1)
  max-eirp: 16
```